### ISSUE TITLE
[feature request] add the ability to create rounded rect

### ISSUE THREAD

**Issue by nagy-nabil** [The-Zagy/zagydraw#6]
might be useful resource to getstarted [src](https://www.scriptol.com/html5/canvas/rounded-rectangle.php)

**Comment by sandstone991** [comment]
I have to disagree with you on that one, definitely this is the place to [start](https://github.com/rough-stuff/rough/issues/196)
also this should come in handy when we want to fill closed shapes (loops drawn with pencil for example)
